docs/logging.md

```diff
@@ -0,0 +1,151 @@
+# How to collect and extract vHive logs to local machine
+
+
+This guide describes how to collect and extract logs in an N-node vHive serverless cluster with Firecracker MicroVMs.
+
+There are a couple of ways to gather and extract logs for vHive to your local machine, whether it is on a single-node cluster or a multi node one. We will present one method to do it. Firstly, if you follow the steps from the [Quickstart guide], logs should already be generated in the `/tmp/vhive-logs` folder at different steps in the workflow. However, sometimes because of running some commands with `screen`, the log folder ends up empty. To mitigate this, you can run the same commands using different `tmux panes`.
+
+> **Note:** `tmux` serves the same role as `screen`, being a terminal multiplexer offering similar features. However, this suggested approach differs a little from the Quickstart's guide approach as we will use different `tmux panes` and not different `tmux sessions`, which are equivalent to terminals.
+
+
+The following section will describe how to set up a Serverless (Knative) Cluster using `tmux`. This section follows the steps from the Quickstart guide with minor modifications for the log generation and running commands without `screen`. We present how to set up a multi-node cluster, however, the same modifications can be used to generate and extract logs from a single-node cluster.
+
+### 1. Setup All Nodes
+**On each node (both master and workers)**, execute the following instructions **as a non-root user with sudo rights** using **bash**:
+1. Open a new tmux session
+   ```bash
+    tmux new -s vhiveSession
+    ```
+3. Clone the vHive repository
+    ```bash
+    git clone --depth=1 https://github.com/vhive-serverless/vhive.git
+    ```
+2. Change your working directory to the root of the repository:
+    ```bash
+    cd vhive
+    ```
+3. Create a directory for vHive logs:
+    ```bash
+    mkdir -p /tmp/vhive-logs
+    ```
```

Comment:
I suggest we merge these three steps into one by using `&&` shell construct. Also, the step order is broken.